tmt/utils.py

```diff
@@ -1066,6 +1066,19 @@ def workdir(self) -> Optional[Path]:
 
         return self._workdir
 
+    @property
+    def clone_dirpath(self) -> Path:
+        """
+        Path for cloning into
+        Used internally for picking specific libraries (or anything else) from cloned repos
+        for filtering purposes, it is removed at the end of relevant step
+        Environment variable TMT_CLONE_DIR is used for the path if provided
+        """
+        try:
+            return Path(os.environ['TMT_CLONE_DIR'])
+        except KeyError:
+            return Path('/tmp/tmt_clone_dir')
```

Comment:
Wouldn't this clash if several `tmt` processes are executed on the same test runner? What about rather using `tempfile` here?